packages/smooth_app/lib/generic_lib/widgets/app_bars/logged_in/app_bar_statistics_card.dart

```diff
@@ -0,0 +1,156 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_svg/svg.dart';
+import 'package:provider/provider.dart';
+import 'package:smooth_app/data_models/preferences/user_preferences.dart';
+import 'package:smooth_app/generic_lib/design_constants.dart';
+import 'package:smooth_app/generic_lib/widgets/app_bars/app_bar_constanst.dart';
+import 'package:smooth_app/pages/preferences/lazy_counter.dart';
+import 'package:smooth_app/themes/smooth_theme.dart';
+import 'package:smooth_app/themes/smooth_theme_colors.dart';
+
+class AppBarStatisticsCard extends StatefulWidget {
+  AppBarStatisticsCard({
+    required this.imagePath,
+    required this.description,
+    required this.lazyCounter,
+    super.key,
+  }) : assert(imagePath.isNotEmpty, 'imagePath must not be empty.'),
+       assert(description.isNotEmpty, 'description must not be empty.');
+
+  final String imagePath;
+  final String description;
+  final LazyCounter lazyCounter;
+
+  @override
+  State<StatefulWidget> createState() => _AppBarStatisticsCardState();
+}
+
+class _AppBarStatisticsCardState extends State<AppBarStatisticsCard> {
+  bool _loading = false;
+
+  @override
+  Widget build(BuildContext context) {
+    final SmoothColorsThemeExtension themeExtension = context
+        .extension<SmoothColorsThemeExtension>();
+
+    return Consumer<UserPreferences>(
```

Comment:
If you need the UserPrefs at the root of the widget, then simply call context.watch.
It will remove a useless callback/builder